spec/models/message_spec.rb

```diff
@@ -60,6 +60,7 @@
                            sender: 'teacher@mumuki.org',
                            read: false,
                            exercise: {bibliotheca_id: problem.bibliotheca_id},
+                           not_actually_a_question: false,
```

Comment:
We should except this one in `import_from_resource_h!` instead of adding it here